sympy/physics/quantum/tensorproduct.py

```diff
@@ -246,9 +246,12 @@ def _eval_expand_tensorproduct(self, **hints):
             if isinstance(args[i], Add):
                 for aa in args[i].args:
                     tp = TensorProduct(*args[:i] + (aa,) + args[i + 1:])
-                    if isinstance(tp, TensorProduct):
-                        tp = tp._eval_expand_tensorproduct()
-                    add_args.append(tp)
+                    c_part, nc_part = tp.args_cnc()
+                    #Check for TensorProduct object: is the one object in nc_part, if any:
+                    #(Note: any other object type to be expanded must be added here)
```

Comment:
```suggestion
                    # (Note: any other object type to be expanded must be added here)
```

Reply:
There should be one spacing between `#` and comment for inline comments
https://peps.python.org/pep-0008/#inline-comments